### PR TITLE
Added survey cards under social cards

### PR DESCRIPTION
**Pull request by linkfang**

### CODE REVIEW

File: content/resources/_index.md [hunk 1]
```diff
@@ -78,3 +78,4 @@ hide_page_title: true
 ## Social Cards{#cards}
 
 - [Jakarta EE Community Cards](/resources/community-cards)
+- [Jakarta EE 2021 Developer Survey Cards](https://f.hubspotusercontent30.net/hubfs/4753786/Completed-Jakarta-Developer-Survey.gif)
```
Comment (ivargrimstad) [inline content/resources/_index.md]: Are we planning more cards for the survey, or only this one?
Maybe the text should be simply "I completed the Jakarta EE Survey"?

Comment (autumnfound) [reply content/resources/_index.md]: We also tend to host images locally to ensure availability, so I would save the gif into the same area I have the community cards and link to it instead!

Comment (linkfang) [reply content/resources/_index.md]: OK, gotcha!

I was thinking use something like "I completed the Jakarta EE Survey", but felt it was more like a sentence than a title of a card. 

So..., how about "Jakarta EE Survey Completion Card"? Will this work? 
I am totally good with "I completed the Jakarta EE Survey", just trying to find maybe a little more proper/accurate name/title for it.

Comment (linkfang) [reply content/resources/_index.md]: Changes have been made:

- Changed the card name to "Completed the Jakarta EE Survey"
- Saved the gif file into the same folder where has all other community cards

**Review by ivargrimstad — APPROVED** [review]

**Pull request merged**